lib/dradis/plugins/burp/gem_version.rb

```diff
@@ -9,7 +9,7 @@ def self.gem_version
       module VERSION
         MAJOR = 3
         MINOR = 11
-        TINY = 0
```

Comment:
this seems like a big change and deserves a minor version bump to 3.12

Reply:
Not sure about the version yet. 
If this is finished before `3.12`, may become a `3.11.1` ?